Dockerfile.fpm

```diff
@@ -1,15 +1,16 @@
 ARG image_tag=latest
+ARG composer=composer
 FROM elifesciences/journal_assets:${image_tag} AS assets
-FROM elifesciences/journal_composer:${image_tag} AS composer
+FROM elifesciences/journal_${composer}:${image_tag} AS composer
 FROM elifesciences/php_7.0_fpm:2f00bfb3e9385f38997f04cea41ffbd3ff9d10a8
 
 ENV PROJECT_FOLDER=/srv/journal
-ENV PHP_ENTRYPOINT=web/app_dev.php
+ENV PHP_ENTRYPOINT=web/app_test.php
```

Comment:
these entry points are a bit weird (`fpm` is oriented to production usage) but are a good start